### PR TITLE
text-decoration : data from MDN

### PR DESCRIPTION
**Pull request by romainmenke**
This adds data for :

- `text-decoration` (as shorthand)
- `text-decoration-color`
- `text-decoration-line`
- `text-decoration-style`

This data is needed to correctly resolve https://github.com/postcss/autoprefixer/issues/1473

### CODE REVIEW

File: copy-mdn.js [hunk 1]
```diff
@@ -221,3 +221,38 @@ writeFileSync(
   JSON.stringify(printColorAdjustData)
 )
 
+const textDecorationColorData = bcdDataToCanIUseData(
+  bcd.css.properties['text-decoration-color'].__compat,
+  'text-decoration-color property'
+)
+writeFileSync(
+  './node_modules/caniuse-db/features-json/css-text-decoration-color.json',
```
Comment (ai) [inline copy-mdn.js:229]: Let’s start using `mdn-` prefix

File: copy-mdn.js [hunk 2]
```diff
@@ -221,3 +221,38 @@ writeFileSync(
...
+  './node_modules/caniuse-db/features-json/css-text-decoration-style.json',
+  JSON.stringify(textDecorationStyleData)
+)
+
+const textDecorationShorthandData = bcdDataToCanIUseData(
+  bcd.css.properties['text-decoration'].shorthand.__compat,
+  'text-decoration shorthand property'
+)
+writeFileSync(
+  './node_modules/caniuse-db/features-json/css-text-decoration_shorthand.json',
```
Comment (romainmenke) [inline copy-mdn.js:256]: @ai Is the underscore ok here?
I followed the convention from caniuse (`mdn-css_properties_text-decoration_shorthand`)

Comment (ai) [reply copy-mdn.js:256]: I used just `mdn-` prefix. But maybe it is a good idea to copy all. https://github.com/browserslist/caniuse-lite/commit/afe96e5ce3ee271e5bf03324521f822771be26e1

What do you think?

**Comment by ai** [comment]
(You will also need to rebase PR since I added my data too)

File: copy-mdn.js (hunk 1 above)
Comment (romainmenke) [reply copy-mdn.js]: Done, thanks

File: copy-mdn.js (hunk 2 above)
Comment (romainmenke) [reply copy-mdn.js]: I've renamed to `mdn-text-decoration-shorthand.json` I think this fits better with `mdn-css-unicode-bidi-isolate-override.json`

**Comment by ai** [comment]
Now let’s wait next `caniuse-db` release

**Pull request merged**